_haikus/Python.md

```diff
@@ -1,10 +1,9 @@
 ---
```

Comment:
Please don't overwrite an existing haiku. Remove this markdown file from the pull request.

Reply:
I'm not sure what happened with this haiku. When I forked the directory it was uncommitted like that. I didn't know what to do with it so I just committed it assuming the author forgot. 
Maybe it can be split up into to haikus ?